build/common.sh

```diff
@@ -1558,18 +1556,25 @@ function kube::release::docker::release() {
 
   for arch in "${archs[@]}"; do
     for binary in "${binaries[@]}"; do
-      local docker_target="${KUBE_DOCKER_REGISTRY}/${binary}-${arch}:${KUBE_DOCKER_IMAGE_TAG}"
-      kube::log::status "Pushing ${binary} to ${docker_target}"
-      "${docker_push_cmd[@]}" push "${docker_target}"
 
-      # If we have a amd64 docker image. Tag it without -amd64 also and push it for compatibility with earlier versions
-      if [[ ${arch} == "amd64" ]]; then
-        local legacy_docker_target="${KUBE_DOCKER_REGISTRY}/${binary}:${KUBE_DOCKER_IMAGE_TAG}"
+      # Temporary fix. hyperkube-arm isn't built in the release process, so we can't push it
+      # This if statement skips the push for hyperkube-arm
```

Comment:
Some lines up, it says that `kube-apiserver`, `kube-controller-manager`, `kube-scheduler`, `kube-proxy` **and** `hyperkube` should be pushed.
Docker images that are `FROM scratch` (`kube-apiserver`, `kube-controller-manager`, `kube-scheduler`, `kube-proxy`) are built for ARM on `amd64` hosts right now, but `hyperkube` needs special treatment (I will fix this soon) and isn't automatically build in the meantime for ARM